RF24.cpp

```diff
@@ -1188,7 +1189,8 @@ void RF24::stopListening(void)
         powerUp();
     }
 #endif
-    write_register(RX_ADDR_P0, pipe0_writing_address, addr_width);
+    write_register(TX_ADDR, txAddress, addr_width);
+    write_register(RX_ADDR_P0, txAddress, addr_width);
```

Comment:
This addition to `stopListening()` causes an increase in compile size for the scanner example. But, that an edge case where `openWritingPipe()` is never actually used.